contracts/src/libraries/EigenDACertVerificationV2Lib.sol

```diff
@@ -60,26 +55,19 @@ library EigenDACertVerificationV2Lib {
     /// @param blobQuorumsBitmap The bitmap of blob quorums
     error RequiredQuorumsNotSubset(uint256 requiredQuorumsBitmap, uint256 blobQuorumsBitmap);
 
-    /// @notice Thrown when a relay key is not set
-    /// @param relayKey The relay key that is not set
-    error RelayKeyNotSet(uint32 relayKey);
-
     /// @notice Status codes for certificate verification results
     enum StatusCode {
         SUCCESS, // Verification succeeded
         INVALID_INCLUSION_PROOF, // Merkle inclusion proof is invalid
-        INVALID_THRESHOLD_PERCENTAGES, // Threshold percentages are invalid
         SECURITY_ASSUMPTIONS_NOT_MET, // Security assumptions not met
         BLOB_QUORUMS_NOT_SUBSET, // Blob quorums not a subset of confirmed quorums
-        REQUIRED_QUORUMS_NOT_SUBSET, // Required quorums not a subset of blob quorums
-        RELAY_KEY_NOT_SET // Relay key not set
+        REQUIRED_QUORUMS_NOT_SUBSET // Required quorums not a subset of blob quorums
 
     }
 
     function verifyDACertV2(
         IEigenDAThresholdRegistry eigenDAThresholdRegistry,
         IEigenDASignatureVerifier signatureVerifier,
-        IEigenDARelayRegistry relayRegistry,
         BatchHeaderV2 memory batchHeader,
         BlobInclusionInfo memory blobInclusionInfo,
         NonSignerStakesAndSignature memory nonSignerStakesAndSignature,
```

Comment:
on second thought this changes the function signature, and hence the abi of the contract (hence the `!` in the PR title I believe?)

Wondering whether we shouldn't at least for now keep the function signature the same even if we don't use the arguments. Until we've settled on a way to version/upgrade this contract.

Reply:
it does not change the ABI for the cert verifier. It only changes the inputs to the library. The CertVerifier itself would just no longer have a relay registry getter, and I highly doubt that this was used in the first place. (Because there's no reason to use it from the CertVerifier!)